src/content/blog/2025-09-05-memorable-wallet-addresses-custom-domain.md

```diff
@@ -0,0 +1,248 @@
+---
+title: "Memorable wallet addresses on your own domain"
+description: "If you own a domain, you can set your Open Payments wallet address to be the same as your domain!"
+date: 2025-09-05
+slug: memorable-wallet-addresses-custom-domain
+authors:
+  - Sid Vishnoi
+author_urls:
+  - https://sidvishnoi.com?ref=ilf_engg_blog
+tags:
+  - Open Payments
+  - Web Monetization
+---
+
+Wallet addresses are meant to be easy to remember or identify, unless your wallet provider chooses them for you. The address might include a long subdomain or even a random series of numbers and characters. But did you know that if you own a domain, you can set your wallet address to be the same as your domain?
+
+So, instead of `https://ilp.wallet.example/12345432/usd`, you can have `$mywebsite.com` as your wallet address (well, technically, a [payment pointer](https://paymentpointers.org/))! It's as easy as pie to remember.
+
+![Illustration showing turning long wallet addresses to custom domain payment pointers](/developers/img/blog/2025-09-05/memorable-wallet-addresses-on-own-domain.png)
+
+You can change your wallet without telling anyone about a new address. And not to forget the branding that comes with your domain - you can share it with people to request money with Open Payments, and it's handy when sending someone money as well. The [Web Monetization extension](https://github.com/interledger/web-monetization-extension) also supports these custom wallet addresses.
```

Comment:
```suggestion
You can change your wallet provider without telling anyone about a new address. And not to forget the branding that comes with your domain - you can share the address with people to request money with Open Payments, and it's handy when sending someone money as well. The [Web Monetization extension](https://github.com/interledger/web-monetization-extension) also supports custom wallet addresses.
```

Reply:
Maybe this section can be reworked a bit. I'm not sure what branding has to do with requesting/sending money with Open Payments.